packages/core/admin/admin/src/content-manager/history/pages/History.tsx

```diff
@@ -99,32 +103,35 @@ const HistoryPage = () => {
     return <Navigate to="/content-manager" />;
   }
 
-  if (isLoadingDocument || isLoadingLayout || versionsResponse.isLoading) {
+  if (isLoadingDocument || isLoadingLayout || versionsResponse.isFetching) {
```

Comment:
- Not sure this did the trick, I still have the issue with the most recent version not always being the selected version

https://github.com/strapi/strapi/assets/26598053/6eee3a88-7121-4fe5-b53e-60d18dcdf6a3